Hotfix to add db backup task (already deployed to production) [Merge 2nd to main]

ianheggie-oaf opened the pull request:
Hotfix/1373 add db backup task - related issue:
* https://github.com/openaustralia/morph/issues/1373

Includes PR:
* https://github.com/openaustralia/morph/pull/1397

ianheggie-oaf committed:
Add zstd package for efficient compression

ianheggie-oaf committed:
Add db:backup and compress all backups

ianheggie-oaf committed:
Remove unused methods that had already been refactored elsewhere

Br3nda commented:
@ianheggie-oaf can you push branches and make PRs from the openaustralia remote, so we can resolve git conflicts when you're not available.

ianheggie-oaf committed:
Fix github Rubocop offenses

ianheggie-oaf committed:
Decrease min code coverage required by 1%

ianheggie-oaf commented:
@Br3nda and @jattree - I have changed this to team based and added team-morph as well as code-reviewers since @jattree is not in code-reviewers group) - FYI - I have requested membership  for code-reviewers

Br3nda committed:
Merge branch 'main' into hotfix/1373-add_db_backup_task
